### ISSUE TITLE
Error occurs after updating

### ISSUE THREAD

**Issue by ranranking** [Shougo/deoplete.nvim#257]
# Problems summary
I have this error when using deoplete:
```vim
Error detected while processing function <SNR>47_completion_begin:
line   26:
E121: Undefined variable: g:deoplete#_channel_id
Press ENTER or type command to continue
Error detected while processing function <SNR>47_completion_begin:
line   26:
E116: Invalid arguments for function rpcnotify
Press ENTER or type command to continue
```

This is my minimum vimrc:
```vim
  if &compatible
    set nocompatible endif
  endif
  set runtimepath+=~/.vim/dein/repos/github.com/Shougo/dein.vim
  let bundleExists = 1

  if dein#load_state(expand('~/.vim/dein'))
    call dein#begin(expand('~/.vim/dein'))

    call dein#add('Shougo/dein.vim', {'rtp':''})
    
    call dein#add('Shougo/deoplete.nvim', {'on_i': 1})

    call dein#end()

    call dein#save_state()
  endif

  filetype plugin indent on

  let g:deoplete#enable_at_startup=1
```

## Environment Information

 * OS: Mac 10.11.1
 * Neovim version: v0.1.4-156-g8d12e28

**Comment by bugeats** [comment]
I'm seeing the same:

```
Error detected while processing function <SNR>154_completion_begin:
line   26:
E121: Undefined variable: g:deoplete#_channel_id
Error detected while processing function <SNR>154_completion_begin:
line   26:
E116: Invalid arguments for function rpcnotify
no notification handler registered for "/Users/chadwick/.config/nvim/plugged/deoplete.nvim/rplugin/python3/deoplete:function:_deoplete"
Error detected while processing function <SNR>154_completion_begin:
line   26:
E121: Undefined variable: g:deoplete#_channel_id
Error detected while processing function <SNR>154_completion_begin:
line   26:
E116: Invalid arguments for function rpcnotify
```

- Mac OS X 10.11.4
- NVIM v0.1.4-156-g8d12e28

My [vimrc here](https://github.com/bugeats/dotfiles)

**Comment by tweekmonster** [comment]
Can you guys confirm your python version by running this in nvim:
`:python3 import sys; print(sys.version)`

If it shows as Python 3.x.x, you'll need to get the RPC host's logs too.  Instructions are detailed here: https://github.com/Shougo/deoplete.nvim/blob/master/.github/ISSUE_TEMPLATE.md

**Comment by Shougo** [comment]
https://github.com/neovim/neovim/pull/4599

**Comment by tweekmonster** [comment]
I was just about to reference that after I saw the messages in Gitter 👍

**Comment by ranranking** [comment]
@Shougo Thank you! I will wait until the PR is merged

**Comment by bugeats** [comment]
Fixed in `NVIM v0.1.4-174-gba9bdb3`.
